core/templates/dev/head/services/contextual/UrlService.js

```diff
@@ -48,6 +48,9 @@ oppia.factory('UrlService', ['$window', function($window) {
     },
     getTopicIdFromUrl: function() {
       var pathname = this.getPathname();
+      if (pathname.split('/')[1] !== 'topic_editor') {
```

Comment:
Check for length of split etc. -- have the code here be similar to getStoryIdFromUrl.

Reply:
Done.